packages/filter/src/components/tag/TagCondition.tsx

```diff
@@ -66,6 +75,10 @@ export const TagCondition: React.FC<ITagConditionProps> = ({
         <TagLabel>Conditions</TagLabel>
         <Select
           shouldFitContainer
+          appearance="filters-tag-condition"
+          menuPortalTarget={ document.body }
+          styles={ tagConditionSelectStyles }
+          closeMenuOnScroll={ (event: Event) => event.target === tagConditionsRef.current }
```

Comment:
did you know that you can just pass the `true` value here? have you tried this?

Reply:
Yes, but it doesn't work.